frontend/src/metabase/visualizations/components/LeafletMarkerPinMap.jsx

```diff
@@ -1,22 +1,23 @@
 import L from "leaflet";
 import _ from "underscore";
 
+import { getSubpathSafeUrl } from "metabase/lib/urls";
 import { isPK } from "metabase-lib/v1/types/utils/isa";
 
 import LeafletMap from "./LeafletMap";
 
-const MARKER_ICON = L.icon({
-  iconUrl: "app/assets/img/pin.png",
-  iconSize: [28, 32],
-  iconAnchor: [15, 24],
-  popupAnchor: [0, -13],
-});
-
 export default class LeafletMarkerPinMap extends LeafletMap {
   componentDidMount() {
     super.componentDidMount();
 
     this.pinMarkerLayer = L.layerGroup([]).addTo(this.map);
+    this.pinMarkerIcon = L.icon({
+      iconUrl: getSubpathSafeUrl("app/assets/img/pin.png"),
+      iconSize: [28, 32],
+      iconAnchor: [15, 24],
+      popupAnchor: [0, -13],
+    });
+
```

Comment:
Decided to put it here, because:
- ideally, it should be created only once
- we should not call `getSubpathSafeUrl` while the root of the module is evaluated, because at that moment `api.basename` is empty